src/bindings/python/python.cc

```diff
@@ -93,6 +93,9 @@ struct simplified_actor_t : public valhalla::tyr::actor_t {
   std::string centroid(const std::string& request_str) {
     return valhalla::tyr::actor_t::centroid(request_str, nullptr, nullptr);
   };
+  std::string status() {
+    return valhalla::tyr::actor_t::status("", nullptr, nullptr);
```

Comment:
We need to pass the request string here for the verbose flag

Reply:
good catch!